.teamcity/Gradle_Check/configurations/PerformanceTest.kt

```diff
@@ -25,25 +28,44 @@ class PerformanceTest(model: CIBuildModel, type: PerformanceTestType, stage: Sta
         }
     }
 
+    requirements {
+        // We observed some relocatability issues with its inputs and it is a long running job
+        doesNotContain("teamcity.agent.name", "ec2")
```

Comment:
This requirement is already set by ``applyPerformanceTestSettings`

Reply:
Good to know. Done.